src/docs/reusable-code-and-services/reusable-services-list.md

```diff
@@ -85,15 +87,15 @@ You can get an unrestricted APIkey for use in government applications by opening
 CI/CD Pipeline Templates and GitHub Actions templates.
 
 - OpenShift Pipeline (aka Tekton) examples:
-    - Installation guide on how to setup pipeline on either your local workspace or from a docker container
-    -  React app build and deploy
-    -  Using builda to build and push an image
-    -  Build and deploy application via helm
-    -  Maven build
-    -  Codeql scan
-    -  Sonar scan
-    -  Trivy scan
-    -  Owasp scan
+  - Installation guide on how to setup pipeline on either your local workspace or from a docker container
+  -  React app build and deploy
+  -  Using builda to build and push an image
+  -  Build and deploy application via helm
+  -  Maven build
+  -  Codeql scan
+  -  Sonar scan
+  -  Trivy scan
+  -  Owasp scan
```

Comment:
```suggestion

  - Installation guide on how to setup pipeline on either your local workspace or from a docker container
  -  React app build and deploy
  -  Using builda to build and push an image
  -  Build and deploy application via helm
  -  Maven build
  -  Codeql scan
  -  Sonar scan
  -  Trivy scan
  -  Owasp scan
```
Is a line break needed here for this list to work?

Reply:
Not in this case where it is a nested list. 

There is a line break between the `CI/CD Pipeline Templates and GitHub Actions templates` intro sentence and the first element `- OpenShift Pipeline (aka Tekton) examples:` which is needed.

But for a list, which has a sublist, the extra space is not needed between the elements. In this case the `- OpenShift Pipeline (aka Tekton) examples:` element and the sub element `- Installation guide on how to setup pipeline on either your local workspace or from a docker container` does NOT need a line break between them.